astro/src/content/docs/apis/users.mdx

```diff
@@ -96,6 +98,17 @@ This API is used to retrieve the information about a single User. You can use th
   </APIField>
 </APIBlock>
 
+<XFusionauthTenantIdHeaderScopedOperation />
+
+<API method="GET" uri="/api/user?loginId={loginId}&loginIdTypes={loginIdTypes}" authentication={["api-key"]} title="Retrieve a User by login Id and login Id types"/>
```

Comment:
Do we need to explain how to pass an array anywhere? (just repeat the query param)

Reply:
You could maybe just say `loginIdTypes=type1[&loginIdTypes=type2][&loginIdTypes=type3]

Reply:
Done.